addons/common/functions/fnc_execRemoteFnc.sqf

```diff
@@ -20,6 +20,8 @@
  */
 #include "script_component.hpp"
 
+ACE_DEPRECATED("ace_common_fnc_execRemoteFnc","3.6.0","ace_common_fnc_globalEvent");
```

Comment:
While I agree with you @commy2, we should go with 2 updates before a deprecated function is removed. Target 3.7.0 for removal of the `execRemoteFnc` function - we should get rid of all the usages within ACE in 3.5.0 however.